Library/Formula/pcre2.rb

```diff
@@ -1,6 +1,6 @@
 class Pcre2 < Formula
   desc "Perl compatible regular expressions library with a new API"
-  homepage "http://www.pcre.org/"
+  homepage "https://pcre.org/"
```

Comment:
The `www.`-less URL is fine, but sadly the https:// authentication fails with a domain mismatch.